hummingbot/connector/exchange/trading212/trading212_api_order_book_data_source.py

```
@@ -0,0 +1,57 @@
+    from .trading212_exchange import Trading212Exchange
+
+
+class Trading212APIOrderBookDataSource(OrderBookTrackerDataSource):
+    _logger: Optional[HummingbotLogger] = None
+
+    def __init__(self, trading_pairs: List[str], connector: "Trading212Exchange", api_factory: WebAssistantsFactory):
+        super().__init__(trading_pairs)
+        self._connector = connector
+        self._api_factory = api_factory
+
+    async def get_last_traded_prices(self, trading_pairs: List[str], domain: Optional[str] = None) -> Dict[str, float]:
```

Ruff (0.13.1)

Unused method argument: `domain`

(ARG002)